src/test/java/GPT/EventTest.java

```diff
@@ -0,0 +1,47 @@
+package GPT;
+
+import org.junit.jupiter.api.Test;
```

Comment:
Would it better to break a new line here? To keep it consistent with [src/test/java/GPT/DeadlineTest.java](https://github.com/nus-cs2103-AY2425S1/ip/pull/197/files#diff-208c4ea4e8815989c3c5aea536ec0e1460974c8368bfc50fea4c2b91aa976f21). (or vice versa).

Reply:
Thank you for the suggestion! I’ve updated the code and fixed the line break for consistency